src/app/modules/item/pages/item-details/item-details.component.html

```diff
@@ -18,22 +18,35 @@
         [routerLink]="'./'"
         routerLinkActive #contentTab="routerLinkActive"
         [routerLinkActiveOptions]="{exact: true}"
-        [class.active]="contentTab.isActive" i18n>
+        [class.active]="contentTab.isActive" i18n
+        [hidden]="contentTab.isActive && taskTabs.length > 0"
+      >
         Content
       </a>
+      <ng-container *ngIf="contentTab.isActive && taskTabs.length > 0">
+        <a
+          *ngFor="let tab of taskTabs"
+          class="nav-tab-item cursor-pointer"
+          [class.active]="contentTab.isActive && tab.active"
+          (click)="tab.setActive()"
+        >
+          {{ tab.name }}
+        </a>
+      </ng-container>
       <a
         *ngIf="!(watchedGroup$ | async) || itemData.item.permissions.canWatch !== 'none' || !contentTab.isActive"
         class="nav-tab-item"
         [routerLink]="'./progress'"
         routerLinkActive #progressTab="routerLinkActive"
-        [class.active]="progressTab?.isActive">
+        [class.active]="progressTab?.isActive"
+      >
         <i class="fa fa-chart-line"></i>
         &nbsp;
         <span i18n>Progress</span>
       </a>
     </nav>
     <div class="bg-white">
-      <alg-item-content *ngIf="contentTab.isActive" [itemData]="itemData"></alg-item-content>
+      <alg-item-content *ngIf="contentTab.isActive" [itemData]="itemData" (tabs)="taskTabs = $event"></alg-item-content>
```

Comment:
* I prefer using a intermediary "callback" function instead of changing variables directly in the template (may be harder to debug, to follow and (not here) we may lose type checking)
* Outputs generally have, as name, the event which happened, considering the value as the argument of this change. "tabs" is a bit short/unclear for an event. Suggestion: `taskTabsChange`